front/pages/w/[wId]/developers/api-keys.tsx

```diff
@@ -196,28 +197,30 @@ export function APIKeys({
               />
             </NewDropdownMenuTrigger>
             <NewDropdownMenuContent>
-              {groups
-                .sort((a, b) => {
-                  // Put global groups first
-                  if (a.kind === "global" && b.kind !== "global") {
-                    return -1;
-                  }
-                  if (a.kind !== "global" && b.kind === "global") {
-                    return 1;
-                  }
+              <ScrollArea className="h-[300px]">
```

Comment:
Note: the ScrollArea viewport needs a fixed height

Reply:
can we make 400? 300 seems small?

Reply:
Set it to 300 as this is the value we have in another places, but we can definitely change that